test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -274,7 +300,8 @@ private static void runTest8() throws Exception {
     private static boolean actRB2 = false;
     private static boolean actRB3 = false;
```

Comment:
These three boolean variables need to be declared volatile, the value is modified on the EDT but their value is read on the main thread.